mmseg/apis/train.py

```diff
@@ -100,20 +101,23 @@ def train_segmentor(model,
     data_loaders = [build_dataloader(ds, **train_loader_cfg) for ds in dataset]
 
     # put model on gpus
+    cfg.device = get_device()
```

Comment:
I think `get_device` is repeated in apis/train.py and tools/train.py, so I suggest deleting this line and just getting device in tools/train.py.
```suggestion
```